IFC4x3/Properties/i/Identifier_3CVqGHgfj9xui_$tglS6m2/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Identifier_3CVqGHgfj9xui_$tglS6m2" Name="Identifier" UniqueId="cc7f4411-aa9b-49ef-8b3e-ff7aaf706c02" PrimaryDataType="IfcIdentifier" />
```

Comment:
This seems more like `IfcRoot.GlobalId` i. e. the sample id of `IfcGeoScienceElement` and not the material.

Reply:
Done.